x/txfees/keeper/hooks.go

```diff
@@ -21,15 +21,20 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 		
 			feetoken, err := k.GetFeeToken(ctx, coin.Denom)
 			if err == nil {
```

Comment:
Can we switch this to `if err != nil { continue }` syntax?

This is definitely correct, but would prefer to keep more in line with the golang standard syntax around errors. Sorry I'm being a human linter :sweat_smile: